zubhub_backend/zubhub/creators/views.py

```diff
@@ -74,7 +74,7 @@ def get(self, request, format=None):
             return Response({'detail': _('Your account is active.')}, status=status.HTTP_200_OK)
         else:
             return Response({
-                'detail': _("Account doesn't exist, is inactive or has been deleted.")}, status=status.HTTP_200_OK)
+                'detail': _("Account doesn't exist, is inactive or has been deleted.")}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
I think status code 403 or 401 will be more suitable. Since this is related to access for the user.

Reply:
Yeah, you are right. I have corrected that.

Reply:
On the contrary, I believe 404 is the right status code to use here. Here `AccountStatusAPIView` treats user accounts like a resource and is simple trying to verify if the account exists or not. The authentication process here is simple there to help achieve that end goal, so unauthorized users are allowed to access this endpoint. Sorry for the confusion @khabdrick

Reply:
No problem @NdibeRaymond 
Based on your comment on the pr, should I close this?

Reply:
I've removed the comment. I saw that you need this for the test is that correct?

Reply:
Yes, you are correct.

Reply:
ok this will be merged soon to allow you continue with the test